### DISCUSSION TITLE
[Feature] Import from a server-local volume

### DISCUSSION THREAD

**Discussion by devinrsmith** [immich-app/immich#6220] [Feature Request]
### The feature

I'd like to be able to do a bulk import into Immich from a server-local volume / disk without needing to go through the Upload API (ie, bulk CLI). Ideally, the workflow would feel similar to how creating an external library works right now, except the images would be imported into the default library. My main use case for this is for initial setups with large image sets. I'm currently running Immich as a TrueNAS Scale app, and it's easy for me to mount a ZFS dataset volume that the Immich server could have native disk access to.

Alternatively, a workflow that migrates from an external library to the default library would serve this use case as well. (I imagine a "migrate" in this scenario would be importing all of the external volume assets into the default library and then "deleting" the external library.)

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#1699
